src/Events/Events.ts

```diff
@@ -312,6 +451,144 @@ class Events {
         return Requests.post(route, data, query, options);
     }
 
+    public static createGroup (event_id : string, data : object,query? : object | null,options? : object |null  ) {
+        let route = this.routeCreateGroup.route.replaceAll('{id}',event_id);
+        
+        return Requests.post(route,data,query,options);
+    } 
+    public static updateGroup(event_id : string, data : object, query? : object | null, options? : object | null) {
+        let route = this.routeUpdateGroup.route.replaceAll('{id}', event_id);
+        return Requests.put(route, data, query, options);
+    }
+    public static listGroup(event_id : string,  query? : object | null, options? : object | null) {
+        let route = this.routeListGroup.route.replaceAll('{id}', event_id);
+
+        return Requests.get(route, query, options);
+    }
+    public static ViewGroup(event_id : string,  query? : object | null, options? : object | null) {
+        let route = this.routeViewGroup.route.replaceAll('{id}', event_id);
+
+        return Requests.get(route, query, options);
+    }
+    public static deleteGroup(data : object, query? : object | null, options? : object | null) {
+        return Requests.post(this.routeDeleteGroup.route, data, query, options);
+    }
+    public static registerAttendee(data : object, query? : object | null, options? : object | null) {
+        return Requests.post(this.routeRegisterAttendee.route, data, query, options);
+    }
+    public static listTickets(event_id : string,  query? : object | null, options? : object | null) {
+        let route = this.routeListTickets.route.replaceAll('{id}', event_id);
+
+        return Requests.get(route, query, options);
+    }
+    public static hasTicket(event_id : string,  query? : object | null, options? : object | null) {
```

Comment:
The account id is required in the parameter, and has to be replaced in the route.